packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -379,14 +423,27 @@ class ProxyController {
         }
     }
 
-    private async reportError(error: AxiosError, url: string, config: ApplicationConstructedProxyConfiguration, errorMessage: string, logCtx: LogContext) {
+    private async reportError(
+        error: AxiosError,
+        url: string,
+        config: ApplicationConstructedProxyConfiguration,
+        errorContent: string | Record<string, string>,
+        logCtx: LogContext
+    ) {
         const safeHeaders = proxyService.stripSensitiveHeaders(config.headers, config);
-        await logCtx.error(`${error.request?.method.toUpperCase()} ${url} failed with status '${error.response?.status}'`, {
-            code: error.response?.status,
-            url,
-            error: new Error(errorMessage),
-            requestHeaders: safeHeaders,
-            responseHeaders: error.response?.headers
+        await logCtx.http(`${error.request?.method.toUpperCase()} ${url} failed with status '${error.response?.status}'`, {
+            meta: {
+                content: errorContent
+            },
+            request: {
+                method: config.method,
+                url,
+                headers: safeHeaders
+            },
+            response: {
+                code: error.response?.status || 500,
+                headers: error.response?.headers as Record<string, string>
+            }
```

Comment:
note we can also modify the client to build the object automatically (like the way I did with `error`) just to reuse the logic

Reply:
Not sure it make sense to tie logs to Axios.